dataService.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-undef */
+var product = require('./src/data/products.json');
+var category = require('./src/data/categories.json');
+
+let p = product.products;
+let c = category.categories;
+
+global.productMap = new Map();
+global.categoryMap = new Map();
+global.combinedProductMap = new Map();
+
+// Assign product ID as key and all other product information as value in map
+function getProducts() {
+  for (food in p) {
```

Comment:
Check for the size of ProductMap here, if it's zero - then load the JSONs and create the map. 
Otherwise just return the productMap

Reply:
Added